### PR TITLE
Add new tech spec

### PR DESCRIPTION
**Pull request by ManfredKarrer**

### CODE REVIEW

**Comment by ManfredKarrer** [comment]
@m52go Not so sure how to deal with the old tech spec. Still not 100% covering all so maybe its good to have it as archive? But feel free to remove it if you think its better...

File: dao-technical-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,470 @@
...
+
+==== Vote reveal phase
+Upon entering the vote reveal phase, each voter automatically publishes their vote reveal tx. There is no fee required for this transaction beside the miner fee. No P2P network data is published.
+
+==== Vote result phase
+In the vote result phase, all nodes calculate the vote result on all proposals and apply the result to the overall BSQ state.
+
+This process uses the hash of the blind vote list from the vote reveal tx to determine the winning majority, in case users had different P2P network payloads of blind votes. The majority is calculated by stake (not merit) of the voters. Only if at least 80% of the network has the same hash is the cycle is valid, otherwise, all proposals and requests are considered rejected.
+
+A proposal is considered accepted if the required quorum and threshold are reached. Quorum is the minimum amount of accumulated vote weight in BSQ which is required. Threshold is the relationship of accepted votes to total votes. It cannot be lower then 50.01%. Each proposal type has different quorum and threshold parameters which can be changed by voting.
```
Comment (m52go) [inline dao-technical-overview.adoc]: This part seems a bit ambiguous to me. If I am understanding this correctly, this is how I would change:

Threshold is the relationship of accepted votes to total votes. Each proposal type has different quorum and threshold parameters which can be changed by voting, but threshold can never be below 50.01%.

Is that right?

**Comment by m52go** [comment]
For the record, I archived the old technical spec. All docs now point to this new doc instead of the old one.

File: dao-technical-overview.adoc (hunk 1 above)
Comment (ManfredKarrer) [reply dao-technical-overview.adoc]: Yes better!

Comment (ManfredKarrer) [reply dao-technical-overview.adoc]: I will push the change...

**Comment by m52go** [comment]
ACK

**Pull request merged**